io.github.torbuntu.Leikr.appdata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Something to consider: this file is not specific to Flathub and can be used by other Linux distributions too. Not a requirement though.